e2e/Makefile

```diff
@@ -70,26 +73,15 @@ logs:
 	rm -rf logs
 
 .PHONY: stop
-stop: $(KIND) 
+stop:
 	$(KIND) delete cluster --name=moco
 	-docker image rm moco:dev
 	-docker image rm moco-backup:dev
 	-docker image prune -f
 
-$(KIND):
-	mkdir -p ../bin
-	curl -sfL -o $@ https://github.com/kubernetes-sigs/kind/releases/download/v$(KIND_VERSION)/kind-linux-amd64
-	chmod a+x $@
-
-$(KUBECTL):
-	mkdir -p ../bin
-	curl -sfL -o $@ https://dl.k8s.io/release/v$(KUBERNETES_VERSION)/bin/linux/amd64/kubectl
-	chmod a+x $@
-
 $(KUSTOMIZE):
```

Comment:
Since kustomize is already installed via aqua, `make kustomize` is also unnecessary.

Reply:
`kustomize` target is removed in ae2bba6